packages/components/src/ComboBox/src/ComboBox.tsx

```diff
@@ -383,4 +383,7 @@ const _ComboBox = forwardRef(ComboBox) as <T extends object>(
 ) => ReturnType<typeof ComboBox>;
 (_ComboBox as NamedExoticComponent).displayName = "ComboBox";
 
-export { _ComboBox as ComboBox, ListBoxItem as ComboBoxItem, ListBoxSection as ComboBoxSection };
+export const ComboBoxItem = ListBoxItem;
```

Comment:
extra work: 

can we re-export the props as well, like
export type ComboBoxItemProps = ListBoxProps;

and do the same for section?